triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerImpl.scala

```diff
@@ -62,16 +81,46 @@ object TriggerRunnerImpl {
         Behaviors.receiveMessagePartial[Message] {
           case QueryACSFailed(cause) =>
             if (wasStopped) {
-              // Never mind that it failed - we were asked to stop
-              // anyway.
-              Behaviors.stopped;
+              // Report the failure to the server.
+              config.server ! TriggerInitializationFailure(
+                config.triggerId,
+                config.jwt,
+                parent,
+                cause.toString)
```

Comment:
Could we send a `Stop` message to the parent TriggerRunner directly here instead of going through the Server? It looks like the Server doesn't do anything besides stop the TriggerRunner, and we have access to the parent here too.